lib/matplotlib/tests/test_agg.py

```diff
@@ -153,8 +153,3 @@ def test_long_path():
     points = np.random.rand(70000)
     ax.plot(points)
     fig.savefig(buff, format='png')
-
-
-if __name__ == "__main__":
```

Comment:
the pytest equivilent is simply "py.test name_of_file" , nose couldn't do that, and needed the extra code, apparently.